bin/cli.js

```diff
@@ -37,6 +37,7 @@ program
 	.option('--stdin', 'read predefined tree from STDIN', false)
 	.option('--warning', 'show warnings about skipped files', false)
 	.option('--debug', 'turn on debug output', false)
+	.option('--allow-circular', 'exit with code 0 when circular dependencies are found', false)
```

Comment:
```suggestion
	.option('--allow-circular', 'not exit with code 1 when circular dependencies are found', false)
```

Describe what's done instead of using its converse. The old description is ambiguous: imagine the case where circular dependencies are found along with other errors. By the old description, madge was meant to exit with `0`, which is untrue.